airbyte-integrations/base-java/javabase.sh

```diff
@@ -4,4 +4,4 @@ set -e
 
 # wrap run script in a script so that we can lazy evaluate the value of APPLICATION. APPLICATION is
 # set by the dockerfile that inherits base-java, so it cannot be evaluated when base-java is built.
-bin/"$APPLICATION" "$@"
+cat <&0 | bin/"$APPLICATION" "$@"
```

Comment:
Shouldn't the integration runner be responsible for this?

Reply:
discussed offline. agreed this was right. also agreed to add a comment explaining what's happening. which i did.